doc/user-guide/user-guide.rst

```diff
@@ -1,74 +1,147 @@
+Since the build requires credentials for the nexus servers, both to access the licensed version of OpenSplice and to pull/push images to the nexus docker registry, it is advisable to use the Jenkins server.
+
+.. note::
+
+  At this point, cycle and revision are still managed manually by users.
+  We are investigating ways of automating the process so users that have to worry about setting them up.
+
+The process to create a new cycle/revision build is to clone the repository and create a ticket branch to work on.
+
+.. prompt:: bash
+
+  git clone https://github.com/lsst-ts/ts_cycle_build.git
+  cd ts_cycle_build/
+  git checkout tickets/DM-XXXXX
+
+Then, update the cycle/revision and software versions as needed.
+If any of the :ref:`core packages <Core-packages>` version is updated, the cycle number must be incremented, otherwise just increment the revision number and updade the software versions.
+
+Both cycle, revision and all software versions are managed by an environment file located in:
+
+.. prompt:: bash
+
+  cycle/cycle.env
+
+At the very top you will find the cycle and revision numbers.
+
+Once this file is updated, you should be ready to start a build.
+For that, commit and push your changes to GitHub.
+Make sure to describe the changes you have made on the commit message and also update the version history.
 
+Once the changes are pushed to GitHub the branch will appear in the `cycle build jenkins job`_.
+Select the branch you are working on in the Branch tab and select Build with Parameters on the left-hand side, which will that you to the :ref:`fig-jenkins-build-with-parameters`.
 
-[This area should give an introduction to users, but at greater depth than the overview on the higher-level page. Some repetition is fine.
-Discussion of high-level control classes (if applicable), primary use cases, links to any useful documentation etc can be included.
-Reference to configurations may also be worthwhile]
+.. _cycle build jenkins job: https://tssw-ci.lsst.org/view/CSC_Docker/job/cycleBuild/
 
-Build Cycle Interface
-======================
+.. figure:: /_static/JenkinsBuildWithParameters.png
+   :name: fig-jenkins-build-with-parameters
+   :target: ../_images/SingleProcessFig.png
+   :alt: Jenkins build with parameters
 
-[This area should link to the XML, then discuss the primary commands/events/telemetry that are expected to be used.
-A full explanation of all possible commands/events/telemetry is beyond the scope of this document.
-However, the description in the XML should be verified to be adequately populated such that a user can find the deeper information if required.]
+   Jenkins build with parameters page.
 
-Example Use-Case
-================
+The build is divided into different steps.
+These steps are designed to maximized reusability of docker layer, minimizing the number of layers in the image and reducing the time it takes to build the system.
+The steps in the build are as follows:
```

Comment:
This doesn't look like all of the components are listed here. Do we want to put the list here? Is it better to just point at the Jenkinsfile for this?

Reply:
OK. I think that looks like everyone.